src/utils/dom.ts

```diff
@@ -6,23 +6,17 @@
 
 import * as d3 from "d3";
 
+export type AnySelection = d3.Selection<any, any, any, any>;
+
 export class DOM {
-  public static transform(s: d3.Selection<any>): d3.Transform;
-  public static transform(s: d3.Selection<any>, x: number, y: number): d3.Selection<any>;
-  public static transform(s: d3.Selection<any>, x?: number, y?: number): any {
-    const xform = d3.transform(s.attr("transform"));
-    if (x == null) {
```

Comment:
Is this logic still captured in your code? Removing the optional args is an API regression (though this whole PR will technically require a major version bump)